src/main/java/com/rarchives/ripme/ripper/rippers/InstagramRipper.java

```diff
@@ -50,6 +50,48 @@ public URL sanitizeURL(URL url) throws MalformedURLException {
         return san_url;
     }
 
+    private List<String> getPostsFromSinglePage(Document Doc) {
+        List<String> imageURLs = new ArrayList<>();
+        JSONArray datas;
+        try {
+            JSONObject json = getJSONFromPage(Doc);
+            if (json.getJSONObject("entry_data").getJSONArray("PostPage")
+                    .getJSONObject(0).getJSONObject("graphql").getJSONObject("shortcode_media")
+                    .has("edge_sidecar_to_children")) {
+
+                datas = json.getJSONObject("entry_data").getJSONArray("PostPage")
+                        .getJSONObject(0).getJSONObject("graphql").getJSONObject("shortcode_media")
+                        .getJSONObject("edge_sidecar_to_children").getJSONArray("edges");
+                for (int i = 0; i < datas.length(); i++) {
+                    JSONObject data = (JSONObject) datas.get(i);
+                    data = data.getJSONObject("node");
+                    if (data.has("is_video") && data.getBoolean("is_video")) {
+                        imageURLs.add(data.getString("video_url"));
+                    } else {
+                        imageURLs.add(data.getString("display_url"));
+                    }
+
+                }
+            } else {
+                JSONObject data = json.getJSONObject("entry_data").getJSONArray("PostPage")
+                        .getJSONObject(0).getJSONObject("graphql").getJSONObject("shortcode_media");
+                if (data.getBoolean("is_video")) {
+                    imageURLs.add(data.getString("video_url"));
+                } else {
+                    imageURLs.add(data.getString("display_url"));
+                }
+
+
+            }
+            return imageURLs;
+
+
```

Comment:
please delete blank lines after `{` and before `}` in lines 61, 73, 83-84, 87-88

Reply:
Done in my last push